app/controllers/global_issue_templates_controller.rb

```diff
@@ -45,17 +43,15 @@ def show
   end
 
   def edit
-    @projects = Project.all
     # Change from request.post to request.patch for Rails4.
     if request.patch? || request.put?
       @global_issue_template.safe_attributes = params[:global_issue_template]
       if @global_issue_template.save
         flash[:notice] = l(:notice_successful_update)
-        redirect_to :action => "show", :id => @global_issue_template.id
-
+        redirect_to action: 'show', id: @global_issue_template.id
       else
         respond_to do |format|
-          format.html { render :action => 'show' }
+          format.html { render action: 'show' }
```

Comment:
__[rails_best_practices]__
* simplify render in controllers [:ref](http://rails-bestpractices.com/posts/2010/12/12/simplify-render-in-controllers/)